components/patient-data/ui/src/main/resources/PhenoTips/AllergiesDataClass.xml

```diff
@@ -63,29 +63,12 @@
     </NKDA>
     <allergies>
       <cache>0</cache>
-      <customDisplay>{{velocity}}
-#if ($xcontext.action == 'edit' || $xcontext.action == 'inline')
-#set ($suggest_script = $xwiki.getURL('PhenoTips.AllergySearch', 'get', 'outputSyntax=plain'))
-{{html wiki=false clean=false}}&lt;input name="xwiki-free-multiselect-suggest-script" type="hidden" value="$!escapetool.xml($suggest_script)"/&gt;&lt;input name="${prefix}${name}" type="hidden" value=""/&gt;{{/html}}
-#end
-#macro (_fms__displayValue $v $counter)
-  #if($xcontext.action == 'edit' || $xcontext.action == 'inline')
-    {{html wiki=false clean=false}}&lt;input class="xwiki-free-multiselect-value" type="text" size="30" name="${prefix}${name}" id="${prefix}${name}_${counter}" value="$!escapetool.xml($v)"/&gt;{{/html}}##
-  #else
-    $!{services.rendering.escape($v, $doc.syntax)}##
-  #end
-#end
-#if ($xcontext.action == 'edit' || $xcontext.action == 'inline')
-(% class="xwiki-free-multiselect"%)
-#end
-#foreach ($v in $value)
-1. #_fms__displayValue($v ${foreach.count})
+      <customDisplay>{{include reference="PhenoTips.PatientSheetMacros" /}}
 
-#end
-#if ((!$value || $value.size() == 0) &amp;&amp; ($xcontext.action == 'edit' || $xcontext.action == 'inline'))
-1. #_fms__displayValue('' 1)
-#end
-{{/velocity}}</customDisplay>
+{{velocity filter="indent"}}{{html wiki=true clean=false}}
+#set ($suggest_script = $xwiki.getURL('PhenoTips.AllergySearch', 'get', 'outputSyntax=plain'))
```

Comment:
Regression: previously entered allergies are no longer suggested. This is because `AllergySearch` returns XML, and the suggest widget was changed to expect JSON by default. It should be simple enough to change `AllergySearch` to return JSON instead.

Actually there's another reason: the URL ends up as `...AllergySearch?outputSyntax=plaininput=asd`, missing an `&` before `input=`.